understanding/consistent-identification.html

```diff
@@ -58,185 +56,139 @@
       <div class="default-grid with-gap leftcol">
          <aside class="box nav-hack sidebar standalone-resource__sidebar ">
             <nav>
-               <header class="box-h ">Page Contents</header>
+               <header class="box-h ">このページの内容</header>
                <div class="box-i">
                   <ul>
-                     <li><a href="#brief">In Brief</a></li>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">意図</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 3.2.4:</span>Consistent Identification (Level AA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 3.2.4:</span>一貫した識別性 (レベル AA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Components that have the <a href="#dfn-same-functionality">same functionality</a> within a <a href="#dfn-set-of-web-pages">set of Web pages</a> are identified consistently.
+                  <p><a href="#dfn-set-of-web-pages">ウェブページ一式</a>の中で<a href="#dfn-same-functionality">同じ機能</a>を有するコンポーネントは、一貫して識別できる。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Actions are more predictable across pages.</dd>
+                  <dt>目標</dt>
+                  <dd>行動は、ページ間でより予測可能になる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Identify repeating functions consistently.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>繰り返し機能を一貫して特定する。</dd>
```

Comment:
「一貫して特定する」が少しわかりにくかったので、以下のように意訳するのはいかがでしょうか。

```suggestion
                  <dd>繰り返される機能を、一貫性のある形で特定する。</dd>
```

Reply:
コメントを受けて改めて考えてみましたが、達成基準を

> Consistent Identification
一貫した識別性

> Components that have the same functionality within a set of web pages are identified consistently.
ウェブページ一式の中で同じ機能を有するコンポーネントは、一貫して識別できる。

と翻訳していることから、むしろ「一貫して識別する」とすべきですか…

上記から「繰り返される機能を一貫して識別する。」としてみましたがどうでしょうか。